sandthorn_event_filter.gemspec

```diff
@@ -28,6 +28,6 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rake"
   spec.add_development_dependency "codeclimate-test-reporter"
 
-  spec.add_runtime_dependency "hamster"
+  spec.add_runtime_dependency "hamster", "=1.0.0"
```

Comment:
https://github.com/hamstergem/hamster/releases

They bumped version from 1.0.0 to 2.0.0 and I don´t know if I trust that 1.x will be compatible if it ever will be released.